blocks/ai-chat/package.json

```diff
@@ -64,19 +65,19 @@
     "icon": "TODO",
     "image": "TODO",
     "protocol": "0.3",
-    "blockEntityType": "http://localhost:3000/@alice/types/entity-type/ai-chat-block/v/6",
+    "blockEntityType": "https://blockprotocol-imqnt3bj2.stage.hash.ai/@alfie/types/entity-type/ai-chat-block/v/1",
     "codegen": {
-      "outputFolder": "src/types",
+      "outputFolder": "src/types/generated",
       "targets": {
-        "generated.ts": [
+        "block-entity.ts": [
           {
             "blockEntityType": true
           },
           {
-            "sourceTypeId": "http://localhost:3000/@alice/types/entity-type/ai-chat-request/v/4"
+            "sourceTypeId": "https://blockprotocol-imqnt3bj2.stage.hash.ai/@alfie/types/entity-type/request-message/v/2"
           },
           {
-            "sourceTypeId": "http://localhost:3000/@alice/types/entity-type/ai-chat-response/v/4"
+            "sourceTypeId": "https://blockprotocol-imqnt3bj2.stage.hash.ai/@alfie/types/entity-type/request-message/v/2"
```

Comment:
(these are the same, I think you meant response for one)

Do you need to specify these types? I'm assuming they're brought in by the `block-entity`

Reply:
If you do want to bring them in perhaps it'd be good to separete it into `request-message.ts` and `request-response.ts`

Reply:
I have fixed the typo and moved them into individual files in https://github.com/hashintel/hash/pull/2299/commits/cc7888d92006160894d3ba8e2fbc28b5b8a06889